2020-08-07 01:16:51.967 [error] <0.2580.0> CRASH REPORT Process <0.2580.0> with 0 neighbours exited with reason: {{{{#{'__exception__' => true,'__struct__' => 'Elixir.Phoenix.Router.NoRouteError',conn => #{'__struct__' => 'Elixir.Plug.Conn',adapter => {'Elixir.Plug.Cowboy.Conn',#{bindings => #{},body_length => 0,cert => undefined,has_body => false,headers => #{<<"accept">> => <<"text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8">>,<<"accept-encoding">> => <<"gzip, deflate">>,<<"accept-language">> => <<"en-US,en;q=0.5">>,<<"connection">> => <<"keep-alive">>,<<"cookie">> => <<...>>,...},...}},...},...},...},...},...},...}
2020-08-07 01:16:51.968 [error] <0.2580.0> Cowboy stream 1 with ranch listener 'Elixir.PuzzlespaceWeb.Endpoint.HTTP' and connection process <0.2579.0> had its request process exit with reason: {{{#{'__exception__' => true,'__struct__' => 'Elixir.Phoenix.Router.NoRouteError',conn => #{'__struct__' => 'Elixir.Plug.Conn',adapter => {'Elixir.Plug.Cowboy.Conn',#{bindings => #{},body_length => 0,cert => undefined,has_body => false,headers => #{<<"accept">> => <<"text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8">>,<<"accept-encoding">> => <<"gzip, deflate">>,<<"accept-language">> => <<"en-US,en;q=0.5">>,<<"connection">> => <<"keep-alive">>,<<"cookie">> => <<"...">>,...},...}},...},...},...},...},...}
2020-08-07 20:08:36.431 [error] <0.3964.0> CRASH REPORT Process <0.3964.0> with 0 neighbours exited with reason: {{{{#{'__exception__' => true,'__struct__' => 'Elixir.Plug.CSRFProtection.InvalidCSRFTokenError',message => <<"invalid CSRF (Cross Site Request Forgery) token, please make sure that:\n\n  * The session cookie is being sent and session is loaded\n  * The request include a valid '_csrf_token' param or 'x-csrf-token' header">>,plug_status => 403},[{'Elixir.Plug.CSRFProtection',call,2,[{file,"lib/plug/csrf_protection.ex"},{line,309}]},{'Elixir.PuzzlespaceWeb.Router',api,2,[]},{'Elixir.PuzzlespaceWeb.Router',...},...]},...},...},...}
2020-08-07 20:08:36.431 [error] <0.3964.0> Cowboy stream 15 with ranch listener 'Elixir.PuzzlespaceWeb.Endpoint.HTTP' and connection process <0.3877.0> had its request process exit with reason: {{{#{'__exception__' => true,'__struct__' => 'Elixir.Plug.CSRFProtection.InvalidCSRFTokenError',message => <<"invalid CSRF (Cross Site Request Forgery) token, please make sure that:\n\n  * The session cookie is being sent and session is loaded\n  * The request include a valid '_csrf_token' param or 'x-csrf-token' header">>,plug_status => 403},[{'Elixir.Plug.CSRFProtection',call,2,[{file,"lib/plug/csrf_protection.ex"},{line,309}]},{'Elixir.PuzzlespaceWeb.Router',api,2,[]},{'Elixir.PuzzlespaceWeb.Router',...},...]},...},...}
2020-08-07 20:10:18.543 [error] <0.3994.0> CRASH REPORT Process <0.3994.0> with 0 neighbours exited with reason: {{{{#{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"expected action/2 to return a Plug.Conn, all plugs must receive a connection (conn) and return a connection, got: %{\"user_input\" => [%{\"buttons\" => [\"M0\"], \"mouse\" => \"DOWN\", \"pos_x\" => 24, \"pos_y\" => 23}, %{\"buttons\" => [\"M1\"], \"mouse\" => \"DRAG\", \"pos_x\" => 45, \"pos_y\" => 22}, %{\"buttons\" => [\"M1\"], \"mouse\" => \"DRAG\", \"pos_x\" => 123, \"pos_y\" => 24}, %{\"buttons\" => [\"M1\"], \"mouse\" => \"DRAG\", \"pos_x\" => 130, ...">>},...},...},...},...}
2020-08-07 20:10:18.544 [error] <0.3994.0> Cowboy stream 6 with ranch listener 'Elixir.PuzzlespaceWeb.Endpoint.HTTP' and connection process <0.3969.0> had its request process exit with reason: {{{#{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"expected action/2 to return a Plug.Conn, all plugs must receive a connection (conn) and return a connection, got: %{\"user_input\" => [%{\"buttons\" => [\"M0\"], \"mouse\" => \"DOWN\", \"pos_x\" => 24, \"pos_y\" => 23}, %{\"buttons\" => [\"M1\"], \"mouse\" => \"DRAG\", \"pos_x\" => 45, \"pos_y\" => 22}, %{\"buttons\" => [\"M1\"], \"mouse\" => \"DRAG\", \"pos_x\" => 123, \"pos_y\" => 24}, %{\"buttons\" => [\"M1\"], \"mouse\" => \"DRAG\", \"pos_x\" => 130, \"p...">>},...},...},...}
2020-08-07 20:11:30.883 [error] <0.3998.0> CRASH REPORT Process <0.3998.0> with 0 neighbours exited with reason: {{{{#{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"expected action/2 to return a Plug.Conn, all plugs must receive a connection (conn) and return a connection, got: %{\"user_input\" => [%{\"buttons\" => [\"Alt\"], \"pos_x\" => 15, \"pos_y\" => 60}]}">>},[{'Elixir.PuzzlespaceWeb.PageController',phoenix_controller_pipeline,2,[{file,"lib/puzzlespace_web/controllers/page_controller.ex"},{line,1}]},{'Elixir.Phoenix.Router','__call__',2,[{file,"lib/phoenix/router..."},...]},...]},...},...},...}
2020-08-07 20:11:30.883 [error] <0.3998.0> Cowboy stream 1 with ranch listener 'Elixir.PuzzlespaceWeb.Endpoint.HTTP' and connection process <0.3997.0> had its request process exit with reason: {{{#{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"expected action/2 to return a Plug.Conn, all plugs must receive a connection (conn) and return a connection, got: %{\"user_input\" => [%{\"buttons\" => [\"Alt\"], \"pos_x\" => 15, \"pos_y\" => 60}]}">>},[{'Elixir.PuzzlespaceWeb.PageController',phoenix_controller_pipeline,2,[{file,"lib/puzzlespace_web/controllers/page_controller.ex"},{line,1}]},{'Elixir.Phoenix.Router','__call__',2,[{file,"lib/phoenix/router.e..."},...]},...]},...},...}
2020-08-07 20:11:33.706 [error] <0.4000.0> CRASH REPORT Process <0.4000.0> with 0 neighbours exited with reason: {{{{#{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"expected action/2 to return a Plug.Conn, all plugs must receive a connection (conn) and return a connection, got: %{\"user_input\" => [%{\"buttons\" => [\"Alt\"], \"pos_x\" => 15, \"pos_y\" => 60}]}">>},[{'Elixir.PuzzlespaceWeb.PageController',phoenix_controller_pipeline,2,[{file,"lib/puzzlespace_web/controllers/page_controller.ex"},{line,1}]},{'Elixir.Phoenix.Router','__call__',2,[{file,"lib/phoenix/router..."},...]},...]},...},...},...}
2020-08-07 20:11:33.708 [error] <0.4000.0> Cowboy stream 2 with ranch listener 'Elixir.PuzzlespaceWeb.Endpoint.HTTP' and connection process <0.3997.0> had its request process exit with reason: {{{#{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"expected action/2 to return a Plug.Conn, all plugs must receive a connection (conn) and return a connection, got: %{\"user_input\" => [%{\"buttons\" => [\"Alt\"], \"pos_x\" => 15, \"pos_y\" => 60}]}">>},[{'Elixir.PuzzlespaceWeb.PageController',phoenix_controller_pipeline,2,[{file,"lib/puzzlespace_web/controllers/page_controller.ex"},{line,1}]},{'Elixir.Phoenix.Router','__call__',2,[{file,"lib/phoenix/router.e..."},...]},...]},...},...}
2020-08-07 20:11:39.444 [error] <0.4002.0> CRASH REPORT Process <0.4002.0> with 0 neighbours exited with reason: {{{{#{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"expected action/2 to return a Plug.Conn, all plugs must receive a connection (conn) and return a connection, got: %{\"user_input\" => [%{\"buttons\" => [\"Alt\"], \"pos_x\" => 15, \"pos_y\" => 60}]}">>},[{'Elixir.PuzzlespaceWeb.PageController',phoenix_controller_pipeline,2,[{file,"lib/puzzlespace_web/controllers/page_controller.ex"},{line,1}]},{'Elixir.Phoenix.Router','__call__',2,[{file,"lib/phoenix/router..."},...]},...]},...},...},...}
2020-08-07 20:11:39.446 [error] <0.4002.0> Cowboy stream 3 with ranch listener 'Elixir.PuzzlespaceWeb.Endpoint.HTTP' and connection process <0.3997.0> had its request process exit with reason: {{{#{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"expected action/2 to return a Plug.Conn, all plugs must receive a connection (conn) and return a connection, got: %{\"user_input\" => [%{\"buttons\" => [\"Alt\"], \"pos_x\" => 15, \"pos_y\" => 60}]}">>},[{'Elixir.PuzzlespaceWeb.PageController',phoenix_controller_pipeline,2,[{file,"lib/puzzlespace_web/controllers/page_controller.ex"},{line,1}]},{'Elixir.Phoenix.Router','__call__',2,[{file,"lib/phoenix/router.e..."},...]},...]},...},...}
2020-08-07 22:50:36.793 [error] <0.4493.0>@Elixir.RPCCoordinator.RPCHandler:-handle_cast/2-fun-0-:76 CRASH REPORT Process <0.4493.0> with 0 neighbours crashed with reason: bad function #Fun<Elixir.STPuzzleCoordinator.1.103840674> in 'Elixir.RPCCoordinator.RPCHandler':'-handle_cast/2-fun-0-'/6 line 76
2020-08-07 22:50:36.793 [error] <0.601.0>@Elixir.RPCCoordinator.RPCHandler:-handle_cast/2-fun-0-:76 Supervisor {<0.601.0>,'Elixir.Task.Supervisor'} had child undefined started with {'Elixir.Task.Supervised',start_link,undefined} at <0.4493.0> exit with reason bad function #Fun<Elixir.STPuzzleCoordinator.1.103840674> in 'Elixir.RPCCoordinator.RPCHandler':'-handle_cast/2-fun-0-'/6 line 76 in context child_terminated
2020-08-07 22:52:29.051 [error] <0.636.0> CRASH REPORT Process <0.636.0> with 0 neighbours exited with reason: {{{{#{'__exception__' => true,'__struct__' => 'Elixir.CaseClauseError',term => <<"M0">>},[{'Elixir.STPuzzleCoordinator',move_desc_to_tuple,1,[{file,"lib/puzzlespace/puzzles/stpuzzle/s_t_puzzle_coordinator.ex"},{line,75}]},{'Elixir.Enum','-map/2-lists^map/1-0-',2,[{file,"lib/enum.ex"},{line,1396}]},{'Elixir.STPuzzleCoordinator',update,3,[{file,"lib/puzzlespace/puzzles/stpuzzle/s_t_puzzle_coordinator.ex"},{line,64}]},{'Elixir.Puzzlespace.Puzzles',update,3,[{file,"lib/puzzlespace/puzzles/pu..."},...]},...]},...},...},...}
2020-08-07 22:52:29.115 [error] <0.636.0> Cowboy stream 7 with ranch listener 'Elixir.PuzzlespaceWeb.Endpoint.HTTP' and connection process <0.606.0> had its request process exit with reason: {{{#{'__exception__' => true,'__struct__' => 'Elixir.CaseClauseError',term => <<"M0">>},[{'Elixir.STPuzzleCoordinator',move_desc_to_tuple,1,[{file,"lib/puzzlespace/puzzles/stpuzzle/s_t_puzzle_coordinator.ex"},{line,75}]},{'Elixir.Enum','-map/2-lists^map/1-0-',2,[{file,"lib/enum.ex"},{line,1396}]},{'Elixir.STPuzzleCoordinator',update,3,[{file,"lib/puzzlespace/puzzles/stpuzzle/s_t_puzzle_coordinator.ex"},{line,64}]},{'Elixir.Puzzlespace.Puzzles',update,3,[{file,"lib/puzzlespace/puzzles/puzz..."},...]},...]},...},...}
2020-08-07 23:40:41.597 [error] <0.605.0> gen_server <0.605.0> terminated with reason: heartbeat_timeout
2020-08-07 23:40:41.598 [error] <0.605.0> CRASH REPORT Process <0.605.0> with 0 neighbours exited with reason: heartbeat_timeout in gen_server:handle_common_reply/8 line 751
2020-08-07 23:40:41.599 [error] <0.603.0> Supervisor {<0.603.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.604.0>, {amqp_params_network,<<"psserver">>,{plaintext,<<"fizzbuzz">>},<<"/">>,"192.168.1.142",5672,0,0,...}) at <0.605.0> exit with reason heartbeat_timeout in context child_terminated
2020-08-07 23:40:41.599 [error] <0.603.0> Supervisor {<0.603.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.604.0>, {amqp_params_network,<<"psserver">>,{plaintext,<<"fizzbuzz">>},<<"/">>,"192.168.1.142",5672,0,0,...}) at <0.605.0> exit with reason reached_max_restart_intensity in context shutdown
2020-08-07 23:41:20.304 [error] <0.859.0>@gen_server:call CRASH REPORT Process <0.859.0> with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.614.0>, {subscribe,{'basic.consume',0,<<"stbridges_response">>,<<>>,false,false,false,false,[]},<0.860.0>}, 60000) in gen_server:call/3 line 223
2020-08-07 23:41:20.306 [error] <0.618.0>@gen_server:call Supervisor {<0.618.0>,'Elixir.Task.Supervisor'} had child undefined started with {'Elixir.Task.Supervised',start_link,undefined} at <0.859.0> exit with reason no such process or port in call to gen_server:call(<0.614.0>, {subscribe,{'basic.consume',0,<<"stbridges_response">>,<<>>,false,false,false,false,[]},<0.860.0>}, 60000) in context child_terminated
2020-08-07 23:41:47.489 [error] <0.865.0>@gen_server:call CRASH REPORT Process <0.865.0> with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.614.0>, {subscribe,{'basic.consume',0,<<"stbridges_response">>,<<>>,false,false,false,false,[]},<0.866.0>}, 60000) in gen_server:call/3 line 223
2020-08-07 23:41:47.489 [error] <0.618.0>@gen_server:call Supervisor {<0.618.0>,'Elixir.Task.Supervisor'} had child undefined started with {'Elixir.Task.Supervised',start_link,undefined} at <0.865.0> exit with reason no such process or port in call to gen_server:call(<0.614.0>, {subscribe,{'basic.consume',0,<<"stbridges_response">>,<<>>,false,false,false,false,[]},<0.866.0>}, 60000) in context child_terminated
2020-08-07 23:42:07.209 [error] <0.903.0>@gen_server:call CRASH REPORT Process <0.903.0> with 0 neighbours exited with reason: no such process or port in call to gen_server:call(<0.614.0>, {subscribe,{'basic.consume',0,<<"stbridges_response">>,<<>>,false,false,false,false,[]},<0.905.0>}, 60000) in gen_server:call/3 line 223
2020-08-07 23:42:07.210 [error] <0.618.0>@gen_server:call Supervisor {<0.618.0>,'Elixir.Task.Supervisor'} had child undefined started with {'Elixir.Task.Supervised',start_link,undefined} at <0.903.0> exit with reason no such process or port in call to gen_server:call(<0.614.0>, {subscribe,{'basic.consume',0,<<"stbridges_response">>,<<>>,false,false,false,false,[]},<0.905.0>}, 60000) in context child_terminated
2020-08-07 23:42:35.156 [error] <0.590.0> CRASH REPORT Process <0.590.0> with 0 neighbours exited with reason: {{{{noproc,{'Elixir.GenServer',call,['Elixir.Puzzlespace.SessionHandler',{check,<<"2Ydw4U4KmJaUxDziby9Ez4+HEL0=">>},5000]}},{'Elixir.PuzzlespaceWeb.Endpoint',call,[#{'__struct__' => 'Elixir.Plug.Conn',adapter => {'Elixir.Plug.Cowboy.Conn',#{bindings => #{},body_length => 0,cert => undefined,has_body => false,headers => #{<<"accept">> => <<"text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8">>,<<"accept-encoding">> => <<"gzip, deflate">>,<<"accept-language">> => <<"...">>,...},...}},...},...]}},...},...}
2020-08-07 23:42:35.604 [error] <0.590.0> Cowboy stream 1 with ranch listener 'Elixir.PuzzlespaceWeb.Endpoint.HTTP' and connection process <0.589.0> had its request process exit with reason: {{{noproc,{'Elixir.GenServer',call,['Elixir.Puzzlespace.SessionHandler',{check,<<"2Ydw4U4KmJaUxDziby9Ez4+HEL0=">>},5000]}},{'Elixir.PuzzlespaceWeb.Endpoint',call,[#{'__struct__' => 'Elixir.Plug.Conn',adapter => {'Elixir.Plug.Cowboy.Conn',#{bindings => #{},body_length => 0,cert => undefined,has_body => false,headers => #{<<"accept">> => <<"text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8">>,<<"accept-encoding">> => <<"gzip, deflate">>,<<"accept-language">> => <<"e...">>,...},...}},...},...]}},...}
2020-08-07 23:42:48.884 [error] <0.592.0> CRASH REPORT Process <0.592.0> with 0 neighbours exited with reason: {{{{noproc,{'Elixir.GenServer',call,['Elixir.Puzzlespace.SessionHandler',{check,<<"2Ydw4U4KmJaUxDziby9Ez4+HEL0=">>},5000]}},{'Elixir.PuzzlespaceWeb.Endpoint',call,[#{'__struct__' => 'Elixir.Plug.Conn',adapter => {'Elixir.Plug.Cowboy.Conn',#{bindings => #{},body_length => 0,cert => undefined,has_body => false,headers => #{<<"accept">> => <<"text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8">>,<<"accept-encoding">> => <<"gzip, deflate">>,<<"accept-language">> => <<"...">>,...},...}},...},...]}},...},...}
2020-08-07 23:42:48.898 [error] <0.592.0> Cowboy stream 2 with ranch listener 'Elixir.PuzzlespaceWeb.Endpoint.HTTP' and connection process <0.589.0> had its request process exit with reason: {{{noproc,{'Elixir.GenServer',call,['Elixir.Puzzlespace.SessionHandler',{check,<<"2Ydw4U4KmJaUxDziby9Ez4+HEL0=">>},5000]}},{'Elixir.PuzzlespaceWeb.Endpoint',call,[#{'__struct__' => 'Elixir.Plug.Conn',adapter => {'Elixir.Plug.Cowboy.Conn',#{bindings => #{},body_length => 0,cert => undefined,has_body => false,headers => #{<<"accept">> => <<"text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8">>,<<"accept-encoding">> => <<"gzip, deflate">>,<<"accept-language">> => <<"e...">>,...},...}},...},...]}},...}
2020-08-07 23:43:27.572 [error] <0.590.0> CRASH REPORT Process <0.590.0> with 0 neighbours exited with reason: {{{{noproc,{'Elixir.GenServer',call,['Elixir.Puzzlespace.SessionHandler',{check,<<"2Ydw4U4KmJaUxDziby9Ez4+HEL0=">>},5000]}},{'Elixir.PuzzlespaceWeb.Endpoint',call,[#{'__struct__' => 'Elixir.Plug.Conn',adapter => {'Elixir.Plug.Cowboy.Conn',#{bindings => #{},body_length => 0,cert => undefined,has_body => false,headers => #{<<"accept">> => <<"text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8">>,<<"accept-encoding">> => <<"gzip, deflate">>,<<"accept-language">> => <<"...">>,...},...}},...},...]}},...},...}
2020-08-07 23:43:27.583 [error] <0.590.0> Cowboy stream 1 with ranch listener 'Elixir.PuzzlespaceWeb.Endpoint.HTTP' and connection process <0.589.0> had its request process exit with reason: {{{noproc,{'Elixir.GenServer',call,['Elixir.Puzzlespace.SessionHandler',{check,<<"2Ydw4U4KmJaUxDziby9Ez4+HEL0=">>},5000]}},{'Elixir.PuzzlespaceWeb.Endpoint',call,[#{'__struct__' => 'Elixir.Plug.Conn',adapter => {'Elixir.Plug.Cowboy.Conn',#{bindings => #{},body_length => 0,cert => undefined,has_body => false,headers => #{<<"accept">> => <<"text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8">>,<<"accept-encoding">> => <<"gzip, deflate">>,<<"accept-language">> => <<"e...">>,...},...}},...},...]}},...}
2020-08-07 23:44:17.795 [error] <0.581.0> CRASH REPORT Process <0.581.0> with 0 neighbours exited with reason: {timeout,{gen_server,call,[<0.584.0>,connect,60000]}} in gen_server:call/3 line 223
2020-08-07 23:44:17.795 [error] <0.441.0> Supervisor 'Elixir.Puzzlespace.Supervisor' had child 'Elixir.STPuzzleCoordinator' started with 'Elixir.STPuzzleCoordinator':initialize() at undefined exit with reason {'EXIT',{{badmatch,{error,{timeout,{gen_server,call,[<0.584.0>,connect,60000]}}}},[{'Elixir.STPuzzleCoordinator',initialize,0,[{file,"lib/puzzlespace/puzzles/stpuzzle/s_t_puzzle_coordinator.ex"},{line,15}]},{supervisor,do_start_child_i,3,[{file,"supervisor.erl"},{line,379}]},{supervisor,do_start_child,2,[{file,"supervisor.erl"},{line,365}]},{supervisor,'-start_children/2-fun-0-',3,[{file,"supervisor.erl"},{line,349}]},{supervisor,children_map,4,[{file,"supervisor.erl"},{line,1157}]},{supervisor,...},...]}} in context start_error
2020-08-07 23:44:17.936 [error] <0.588.0> gen_server <0.588.0> terminated with reason: {socket_error,timeout}
2020-08-07 23:44:22.788 [error] <0.455.0> Supervisor 'Elixir.PuzzlespaceWeb.Endpoint' had child 'Elixir.Plug.Cowboy.Drainer' started with 'Elixir.Plug.Cowboy.Drainer':start_link([{refs,['Elixir.PuzzlespaceWeb.Endpoint.HTTP']}]) at <0.579.0> exit with reason killed in context shutdown_error
2020-08-07 23:44:22.791 [error] <0.439.0> CRASH REPORT Process <0.439.0> with 0 neighbours exited with reason: {{shutdown,{failed_to_start_child,'Elixir.STPuzzleCoordinator',{'EXIT',{{badmatch,{error,{timeout,{gen_server,call,[<0.584.0>,connect,60000]}}}},[{'Elixir.STPuzzleCoordinator',initialize,0,[{file,"lib/puzzlespace/puzzles/stpuzzle/s_t_puzzle_coordinator.ex"},{line,15}]},{supervisor,do_start_child_i,3,[{file,"supervisor.erl"},{line,379}]},{supervisor,do_start_child,2,[{file,"supervisor.erl"},{line,365}]},{supervisor,'-start_children/2-fun-0-',3,[{file,"supervisor.erl"},{line,349}]},{supervisor,...},...]}}}},...} in application_master:init/4 line 138
